server/views/partials/nav.pug

```diff
@@ -0,0 +1,23 @@
+// Navigation
+nav#mainNav.navbar.navbar-expand-lg.bg-secondary.fixed-top.text-uppercase
+  .container
+    a.navbar-brand.js-scroll-trigger(href='#page-top') Start Bootstrap
+    button.navbar-toggler.navbar-toggler-right.text-uppercase.bg-primary.text-white.rounded(type='button', data-toggle='collapse', data-target='#navbarResponsive', aria-controls='navbarResponsive', aria-expanded='false', aria-label='Toggle navigation')
+      | Menu
+      i.fas.fa-bars
+    #navbarResponsive.collapse.navbar-collapse
+      ul.navbar-nav.ml-auto
+        if (!session || !session.userId)
+        else
+          li.nav-item.mx-0.mx-lg-1
+            a.nav-link.py-3.px-0.px-lg-3.rounded(href=`/team`) Team
+        li.nav-item.mx-0.mx-lg-1
+          if (!session || !session.userId)
+            a.nav-link.py-3.px-0.px-lg-3.rounded(href= `/api/auth/github/callback`) Signin
```

Comment:
Ok, I see what you did there... At first I thought this shouldn't work and that there was an issue with it. But. now I realise I just had an extra step in the mix.